Tests/PAML/gen_results.py

```diff
@@ -79,28 +79,28 @@ def baseml(vers=None, verbose=False):
                             test[0] == "nparK" and n in [3, 4]):
                         continue
                     print("\t\tn = {0}".format(n))
-                    ctl_file = os.path.join("Control_files", "baseml",
-                        "{0}{1}.ctl".format(test[0], n))
+                    ctl_file = (os.path.join("Control_files", "baseml",
+                                "{0}{1}.ctl".format(test[0], n)))
                     bml.read_ctl_file(ctl_file)
                     bml.alignment = alignment
                     bml.tree = tree
                     out_file = "{0}{1}-{2}.out".format(test[0], n, version)
-                    bml.out_file = os.path.join("Results", "baseml", test[0],
-                        out_file)
+                    bml.out_file = (os.path.join("Results", "baseml", test[0],
+                                    out_file))
```

Comment:
Does fixing the indentation alone (without adding brackets) make the second line too long?